src/components/Product/StyledProduct.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+const ProductContainer = styled.div`
```

Comment:
Your `Product` component renders all your products or? To have more semantic code you could work with a `ul` element where you map your `products` in a `Product` component that is a `li` element.